app/controllers/posts_controller.rb

```diff
@@ -27,6 +27,11 @@ def create
     end
   end
 
+  def destroy
+    post = Post.find(params[:id])
+    redirect_to user_path(post.author_id) if Post.destroy(params[:id])
```

Comment:
- [ ] As per the guidelines provided by Micoverse please make sure you've used **CanCanCan** for Post authorization in `Post Controller`. 

```ruby
class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```

Reply:
Hi, @arslanbisharat. I'm going to add the load_and_authorize_resource method in the Post Controller. Thanks. :+1:

Reply:
Well resolved 👍🏽